packages/backend/src/persistence/storage/postgres/dto/ConversationRecord.ts

```diff
@@ -5,4 +5,6 @@ export type ConversationRecord = {
     previewMessage: string | null;
     //The time the conversation was last updated
     updatedAt: Date;
+    //This field can be used by the client to store information about the contacts TLD name
```

Comment:
encrypt (0x123.hicony.eth). This address can be used by the client to retrieve the UserProfile from Hicoinies Offchain resolver

Reply:
Might also be used to retrieve profiles faster because the time consuming on-chain lookup could be avoided

Reply:
Is this time reduction the main motivation for storing the encryptedProfileLocation? Isn't 0x123.hicony.eth associated with this conversation anyway?

Reply:
client uses the address as the ID to store the contact. So it has to store the location of the profile as well to know where to find the profile. Since it is not published on ENS